tools/travis_watchdog.sh

```diff
@@ -273,27 +280,27 @@ cd ../../
 case $TEST in
     (misc)
         if [ $EXIT_CODE == 0 ]; then
-            printf "\n\n==============================================================================\n"
-            printf "Running bash end-to-end tests\n"
-            printf "==============================================================================\n"
+            echo "\n\n==============================================================================\n"
+            echo "Running bash end-to-end tests\n"
+            echo "==============================================================================\n"
 
             FLINK_DIR=build-target flink-end-to-end-tests/run-pre-commit-tests.sh
 
             EXIT_CODE=$?
         else
-            printf "\n==============================================================================\n"
-            printf "Previous build failure detected, skipping bash end-to-end tests.\n"
-            printf "==============================================================================\n"
+            echo "\n==============================================================================\n"
+            echo "Previous build failure detected, skipping bash end-to-end tests.\n"
+            echo "==============================================================================\n"
         fi
         if [ $EXIT_CODE == 0 ]; then
-            printf "\n\n==============================================================================\n"
-            printf "Running java end-to-end tests\n"
```

Comment:
if we intend to drop this file anyway soon-ish, why are we making this kind of change?

Reply:
I did not know that there are intentions to drop this file. 
If I'm not mistaken, printf did not produce any outputs on azure?! I don't exactly remember. If this is a big problem for you, I can invest time into this.

Reply:
I didn't realize at this point that you're calling the _travis_ watchdog from the azure scripts ;)

Consider this comment resolved. Do note however that there are plenty of printf statements in the e2e scripts that either have to modified as well or we figure out why printf isn't working on azure.